Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+    return (
+        <img className="card-banner" src="https://ibin.co/3wnC6SgIOJud.png" alt="React Banner"/>
```

Comment:
alt tags 🔥 great to see you using these. Making our applications and sites as accessible as possible through attributes like the alt tag is a great habit to develop.